src/pages/DetailProduct/CommentAndRating/index.jsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+import { Row, Col, Card, Rate } from "antd";
+
+function CommentAndRating({ commentList }) {
+  return commentList.data.map(({ comment, email }, index) => {
+    return (
```

Comment:
```suggestion
  return commentList.data.map(({ comment, email }, index) => (
```
Ngoài ra thì, nếu commentList không có thì hiển thị 1 đoạn text chưa có comment nhé

Reply:
có 1 đoạn text chưa có comment, nhưng pull xử lí comment mới xử lí trường hợp đó